addons/binding/org.openhab.binding.nikohomecontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nikohomecontrol" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd"><!--Niko Home Control Gateway Bridge Type -->
+
+	<bridge-type id="bridge">
+		<label>Niko Home Control Bridge</label>
+		<description>This bridge represents a Niko Home Control IP-interface</description>
+		<config-description>
+			<parameter name="ADDR" type="text" required="false">
+				<label>IP or Host Name</label>
+				<description>Address will be resolved on connection if not configured</description>
+				<advanced>true</advanced>
```

Comment:
Yes, as these are all optional in a typical home network and when running openHab on a machine connected to only one network segment. The documentation explains when these parameters are required. E.g. I don't need these when running openHab from my laptop, but I do when running from my Pi that doubles as a VPN server. On the Pi, discovery wrongly picks the VPN network interface to look for a broadcast address. There is no way to know what network to use unless the user configures one or more of these advanced parameters.